src/main/java/com/google/api/codegen/config/MethodConfig.java

```diff
@@ -74,8 +74,14 @@
 
   public abstract ReleaseLevel getReleaseLevel();
 
+  /**
+   * package-private for internal use.
+   *
+   * @see MethodContext#getLongRunningConfig wraps this call, because there are times where we want
```

Comment:
This will make the whole `wraps this call, because there are times where we want to generate the same API method as an LRO method and also as a non-LRO method.` string a one huge hyperlink in the generated javadoc. (See [getStreamWaitTimeout](http://googleapis.github.io/gax-java/1.40.0/apidocs/) funtion in gax-java documentation for an example of what the `@see` construct is converted into in a javadoc html). You can use `Ctrl+Q` key combination to see rendered javadoc of a method directly from `IntelliJ`.

Did you mean `{@link }` instead?

Reply:
oh thanks!